src/components/navigation/perPage/tests/NavigationPerPage.spec.tsx

```diff
@@ -146,7 +146,20 @@ describe('NavigationPerPage', () => {
       navigationPerPage.setProps(newProps);
       navigationPerPageInstance['handleClick'].call(navigationPerPageInstance, expectedPerPage);
 
-      expect(newProps.onPerPageClick).toHaveBeenCalled();
+      expect(newProps.onPerPageClick).toHaveBeenCalledTimes(1);
+    });
+
+    it('should not call onPerPageClick prop if perPage is identical to currentPerPage', () => {
+      const newProps: INavigationPerPageProps = _.extend({}, NAVIGATION_PER_PAGE_BASIC_PROPS,
+        { onPerPageClick: jasmine.createSpy('onPerPageClick'), currentPerPage: 10 });
+      const expectedPerPage: number = 10;
+
+      expect(() => navigationPerPageInstance['handleClick'].call(navigationPerPageInstance, expectedPerPage)).not.toThrow();
```

Comment:
Can you call `() => navigationPerPageInstance.handleClick(expectedPerPage)` instead?

Reply:
this is like an `as any` by passing for private methods, i copy pasted it from other places in the file

Reply:
In this case i'd prefer the `as any`: `() => (navigationPerPageInstance as any).handleClick(expectedPerPage)` its easier to understand in my opinion